Add cluster-size section to system requirements (LGHP)

lproven opened the pull request:
I don't know why validation is failing, as it validated OK on my local machine.

tbazant commented on xml/deployment_installation.xml:
Capitalize sentences in <title> markup

tbazant commented on xml/deployment_installation.xml:
```diff
@@ -55,6 +55,23 @@
     </itemizedlist>
    </sect3>
+   <sect3 xml:id="sec.caasp.installquick.clusreqs">
+    <title>Cluster requirements</title>
```
Capitalize sentences in <title> markup

cwickert commented:
Now we have two entities:
&lt;!ENTITY master_node   "master node"&gt;
&lt;!ENTITY kmaster       "Master node"&gt;
This is everything but consistent. :-(

lproven commented:
On Fri, 23 Mar 2018 06:36:29 -0700
Christoph Wickert <notifications@github.com> wrote:

> Now we have two entities:
> <!ENTITY master_node "master node">
> <!ENTITY kmaster "Master node">
> This is everything but consistent. :-(

That was me, although it was Tomáš's suggestion. I don't know why the
original entity for "Master node" has a capital letter in it, but I
guess that somewhere, it starts a sentence or something. I thought
that if I removed the capital letter, it might break something
somewhere. So I added a new entity without a capital M.

If that is not right, then what's the correct thing to do? Find where
the original use of the entity that needs a capital, and change it?

-- 
Liam Proven - Technical Writer, SUSE Linux s.r.o.
Corso II, Křižíkova 148/34, 186-00 Praha 8 - Karlín, Czechia
Email: lproven@suse.com - Office telephone: +420 284 241 084

cwickert commented:
I'm not against new entities, but they should be consistent.
